perllib/FixMyStreet/App/Controller/Photo.pm

```diff
@@ -54,8 +54,9 @@ sub index :LocalRegex('^(c/)?([1-9]\d*)(?:\.(\d+))?(?:\.(full|tn|fp))?\.(?:jpeg|
         ($item) = $c->model('DB::Comment')->search( {
             id => $id,
             state => 'confirmed',
+            'problem.state' => [ FixMyStreet::DB::Result::Problem->visible_states() ],
             photo => { '!=', undef },
-        } );
+        }, { prefetch => 'problem' });
```

Comment:
`id`, `state`, and `photo` are column names common to both tables so they'll need to be updated in this query to `'me.id'` etc to avoid an SQL error.